db/migrate/20230411060248_add_columns_to_products.rb

```diff
@@ -0,0 +1,8 @@
+class AddColumnsToProducts < ActiveRecord::Migration[7.0]
+  def change
+    add_column :products, :enabled, :boolean
+    add_column :products, :discount_price, :decimal
+    add_column :products, :permalink, :string
+    change_column_default :products, :enabled, from: true, to: false
```

Comment:
should not this be a separate migration? Also check if this is reversible migration?